packages/runtime/client/rest/src/payout/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './details.js'
+export * from './header.js'
+export * from './status.js'
```

Comment:
This directory is small enough that its contents could live in a single file.

As we talked in a meeting, I recommend watching [this talk](https://www.youtube.com/watch?v=XpDsk374LDE)

Reply:
I didn't like this talk :-) ... 
You see a violation of recipe/ convention and you miss the intent behind...
My Intent here is to get read of infrastructure details and extract the core domain... This is not 100% nailed... I have highlighted that there are concepts of headers and details on all the endpoints and before finding the concepts, I have duplicated logics on each of them...

1. To be sure the API is consistent
2. To have a few examples before making it 1 same logic..

High Cohesion / Low Coupling , Clean code , DDD etc... This talk about the life of a file is so rigid... I can't buy it :-) ... When you do little libraries, you can think that this is the case... But when you do something more complex it blows up... Inconsistencies, your dependencies are a mess, you stay monolithic forever etc...